components/k8s-model-server/testing/README.md

```diff
@@ -0,0 +1,3 @@
+# Building and testing the TF serving image.
+
+work in progress.
```

Comment:
The README should explain how I can trigger this workflow to build a TF Serving image. The README should explain the following

    1. How to set which version of the Kubeflow source repository is used
    2. How to set the GCR registry where the image is published to
    3. Which version of TF serving is used.